### ISSUE TITLE
[Machine Learning] Build a Multi Layer Perceptron (MLP) for Classification using deeplearning4J (DL4J) for Java developers

### ISSUE THREAD

**Issue by osepoo** [section-engineering-education/engineering-education#6609]
### Proposed title of article
 [Machine Learning] Build a Multi Layer Perceptron (MLP) for Classification using deeplearning4J (DL4J) for Java Developers.

### Proposed article introduction
An MLP is a neural network class that consists of a number of neurons, the first of which takes the input while the last layer gives the output.
We are going to create a simple neural network using the deeplearning4J library which is a modern and powerful tool for machine learning for Java developers.
We are going to explore the various attributes of a custom made data set which is simple according to machine learning standards. We will then write the actual Java code that will train the model in relatively small steps and provide a concrete evaluation which will be accurate and precise

### Key takeaways
 
1.	Set up the DL4J library in maven.
2.	Learn how to normalize and split CSV files into training and test evaluation data.
3.	Epochs for setting a required network parameter.
4.	Building and fitting network layers for training the model.
5.	Understand how to analyze output models vivdly to tweak parameters for improved outputs.

### Article quality
This guide will not require a profound knowledge in advanced mathematics and all the other necessary topics for a well grounded ML-expert. It will act as the introductory “Hello World” to novice Java ML developers. It will explain every detail from how to set up the environment to what the ouput will look like with original code.

#### References
N/A

**Comment by LinusMuema** [comment]
Hi @osepoo 

Thanks for submitting your article suggestion. However, we cannot move forward with this article as it has been covered before on [the official docs](https://deeplearning4j.konduit.ai/deeplearning4j/tutorials/quick-start) and there's even a [Youtube video](https://www.youtube.com/watch?v=8EIBIfVlgmU).

Feel free to explore other topics.
